platform/viewer/src/routes/Mode/Mode.tsx

```diff
@@ -250,8 +255,16 @@ export default function ModeRoute({
     hangingProtocolService.setActiveProtocols(hangingProtocol);
     mode?.onModeEnter({ servicesManager, extensionManager, commandsManager });
 
-
     const setupRouteInit = async () => {
+      const filters = Array.from(query.keys()).reduce(
+        (acc: Record<string, string>, val: string) => {
+          if (val !== 'StudyInstanceUIDs') {
+            return { ...acc, [val]: query.get(val) };
+          }
+        },
+        {}
+      );
```

Comment:
can you please write down what this reduce is making as a comment? like as an example in the top of it